src/redux/reducers.js

```diff
@@ -1233,3 +1361,18 @@ export function usernameSubscribers(state = {}, action) {
 export function usernameSubscriptions(state = {}, action) {
   return handleSubs(state, action, ActionTypes.SUBSCRIPTIONS)
 }
+
+export function compensateScroll(state = {}, action) {
```

Comment:
It's not realtime-connected, actually, but more about scrolling for instance height. For now I'd like to this to remain like this.